share/src/main/java/io/zensoft/share/service/PublisherManagerService.java

```diff
@@ -34,5 +35,5 @@
      * <p>
      * @param vacancyDto the object given by listener
      */
-    void getInfo(VacancyDto vacancyDto);
+    void getInfo(VacancyDto vacancyDto, PublisherServiceType publisherServiceType);
```

Comment:
are you sure that `PublisherServiceType` object should be given by listener?
I guess `PublisherServiceType` object should be created during the implementation of the method

Reply:
Yes Controller is done with his work. Listener is dumb